src/index.tsx

```diff
@@ -1,18 +1,3 @@
-import React from 'react';
-import { createRoot } from 'react-dom/client';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import { CRM } from './root/CRM';
 
-const container = document.getElementById('root');
-const root = createRoot(container!);
-
-root.render(
-    <React.StrictMode>
-        <App />
-    </React.StrictMode>
-);
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+export default CRM;
```

Comment:
**Nitcpick**: I'd prefer that we don't use the default export, in order to be able to export more than one component without breaking apps that used the first version of the package.